orchagent/aclorch.cpp

```diff
@@ -4207,6 +4819,19 @@ bool AclOrch::removeAclRule(string table_id, string rule_id)
     return m_AclTables[table_oid].remove(rule_id);
 }
 
+bool AclOrch::removeAclRuleWithEgrSetDscp(string table_id, string rule_id)
```

Comment:
same here as add. Keep the logic in removeAclRule

Reply:
and done